docker-compose.yml

```diff
@@ -442,6 +523,31 @@ services:
     volumes:
       - ./apps/calendars/log:/app/log
 
+  whitehall: &whitehall
+    build: apps/whitehall
+    depends_on:
+      - publishing-api
+      - rummager
+      - static
+      - asset-manager
+      - diet-error-handler
+    environment:
+      DATABASE_URL: mysql2://root:root@mysql/whitehall_development
+      SENTRY_CURRENT_ENV: whitehall
+      SENTRY_DSN: http://user:password@error-handler.dev.gov.uk/123
+      VIRTUAL_HOST: whitehall-admin.dev.gov.uk
+    links:
+      - mysql
+      - nginx-proxy:rummager.dev.gov.uk
+      - nginx-proxy:publishing-api.dev.gov.uk
+      - nginx-proxy:error-handler.dev.gov.uk
+      - nginx-proxy:asset-manager.dev.gov.uk
+      - nginx-proxy:static.dev.gov.uk
+    ports:
+      - "33020:3020"
+    volumes:
+      - ./apps/whitehall/log:/app/log
+
```

Comment:
might be worth adding a comment of a placeholder to indicate where whitehall workers might be, just to indicate they'd be needed for whitehall to work properly.